### ISSUE TITLE
Citing MathProgBase.jl

### ISSUE THREAD

**Issue by klkeys** [JuliaOpt/MathProgBase.jl#98]
I am using MathProgBase.jl in a publication. Is there a standard citation that I may cite? If not, then should I [cite JuMP](https://github.com/JuliaOpt/JuMP.jl#citing-jump)?

**Comment by mlubin** [comment]
Nope, if you'd like to write it we'd love the help :). Citing the github page and/or the "Computing in Operations Research ..." paper would be appropriate.

**Comment by ccoffrin** [comment]
@mlubin do you have thoughts on this approach?

https://guides.github.com/activities/citable-code

**Comment by mlubin** [comment]
@ccoffrin, do you know if there's any way to track who cites a zenodo DOI? The big issue is that it's not indexed by google scholar.

**Comment by ccoffrin** [comment]
I was googling into this and I found that it is indexing some things at least,

https://scholar.google.com/scholar?as_publication=zenodo

For example,

https://scholar.google.com/scholar?cluster=15397132910898847372&hl=en&as_sdt=5,32&sciodt=0,32

Maybe a recent development?  One thing that is a bit of an issue, is that it takes quite a few clicks to get back to the source code.  But still useful.

**Comment by mlubin** [comment]
Once the documentation is redone (https://github.com/JuliaOpt/MathProgBase.jl/issues/169), I'd be open to submitting MathProgBase to [JOSS](http://joss.theoj.org/) to get a DOI. Open to suggestions for other journals. If someone would like to write a full paper and put my name somewhere on it, that's ok also.
